extensions/qotp/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "qotp",
+  "title": "QOTP",
+  "description": "QOTP is a simple tool to generate OTP code",
+  "icon": "qotp.png",
+  "author": "ariesly15",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "qotp-panel",
+      "title": "QOTP",
+      "description": "QOTP is a simple tool to generate OTP code",
+      "keywords": [
+        "otp",
+        "qotp",
+        "ff"
+      ],
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "required": true,
+      "name": "pin",
+      "title": "PinCode",
+      "type": "password",
+      "description": "qotp.corp 平台中用户设置的6位密码",
+      "placeholder": "qotp.corp 平台中用户设置的6位密码"
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.